checklists/alz_checklist.en.json

```diff
@@ -2480,22 +2480,78 @@
         },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Implement SIEM solution to consolidate security data in a central location where it can be correlated across various services.",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "a56888b2-7e83-4404-bd31-b886528502d1",
+            "id": "G03.09",
+            "severity": "High",
+            "graph": "resources| where type == 'microsoft.operationalinsights/workspaces'| extend wsid = properties.customerId| project workspaceResourceId = tolower(id), name, wsid| join (resources| where type == 'microsoft.operationsmanagement/solutions'| where name has 'SecurityInsights'| extend workspaceResourceId = tostring(tolower(properties.workspaceResourceId))| project workspaceResourceId | summarize ResourceCount = count() by workspaceResourceId) on workspaceResourceId| extend RCount = iff(isnull(ResourceCount), 0, ResourceCount)| project-away ResourceCount| extend compliant = (RCount <> 0)",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/connect-data-sources?tabs=azure-portal"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Facilitates proactive threat hunting and investigation via Sentinel - built-in threat detection mechanisms to detect and respond to security threats in real time.",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "edca434d-965f-43b2-84b0-d5d8449b5aaf",
+            "id": "G03.10",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/overview?tabs=azure-portal"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Incorporate security alerts into your incident response system",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "13aad129-a2e5-4099-b738-9918f4d6a917",
+            "id": "G03.11",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/relate-alerts-to-incidents"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Automate the response to security alerts",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "348f74c1-0cae-4475-9f1a-7f0d09887743",
+            "id": "G03.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/automation/automation"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Set up data retention and archive, to make sure your organization retains the data that's important in the long term   ",
```

Comment:
Data retention for what?  Is this a part of G03.02?

Reply:
G03.02 refer specifically to Azure activity logs. This one is for Sentinel. Will this be better:
Define Retention policies when to remove or archive data in Sentinel Log analytic workspace, to make sure your organization retains the data that's important in the long term

Reply:
I think that would be better, but we should try to make it as specific as possible and link back to the CAF specific guidance.

Reply:
The CAF only mention the need to have SOC solution in your env, so its not possible to relate to CAF

Reply:
I think that builds on the topic above then - that if this isn't part of the Landing Zone/Ready approach, it might make sense to go to another checklist.  I am going to look at that a bit more and will aim to have some response to that tomorrow.